showcase/app/templates/page-components/form/text-input.hbs

```diff
@@ -296,10 +296,10 @@
 
   <Shw::Text::H3>Character count</Shw::Text::H3>
 
-  <Shw::Grid @columns="3" as |SG|>
+  <Shw::Grid @columns={{3}} as |SG|>
     <SG.Item @label="Label + Character count">
       <Hds::Form::TextInput::Field
-        @value={{this.defaultText}}
+        @value={{this.values.defaultText}}
```

Comment:
for `MaskedInput` we use `this.fieldValues`, for `TextInput` and `Textarea` we use `this.values`, can we use the same naming for all of them? (probably `fieldValues`, more explicit)